assets/csidriveroperators/kubevirt/07_deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubevirt-csi-driver-operator
+  namespace: openshift-cluster-csi-drivers
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: kubevirt-csi-driver-operator
+  template:
+    metadata:
+      labels:
+        name: kubevirt-csi-driver-operator
+    spec:
```

Comment:
We traditionally run CSI operators (and CSI driver controller pods) on masters, i.e. with the right tolerations and node selector. Why should be this operator different?

Reply:
I'm not sure I follow. Can u give example? As far as I'm concerned, I should do whatever complies with CSO. So just lead me and I'll do it.

Reply:
Just copy nodeSelector and tolerations from AWS EBS operator dir

Reply:
I checked oVirt operator and it is  the same, no?